.gitlab-ci.yml

```diff
@@ -80,6 +80,30 @@ test-linux-stable:                 &test
     - time cargo test --all --release --verbose --locked
     - sccache -s
 
+
+test-node-template:
+  stage:                           test
+  variables:
+    RUST_TOOLCHAIN: stable
+    # Enable debug assertions since we are running optimized builds for testing
+    # but still want to have debug assertions.
+    RUSTFLAGS: -Cdebug-assertions=y
+    TARGET: native
+  tags:
+    - linux-docker
+  only:
+    changes:
+      - node-template/*
+  before_script:
+    - sccache -s
+    - ./scripts/build.sh
```

Comment:
Sorry for being late, but you don't need to call this before building the node template.